pages/signin/index.module.scss

```
@@ -0,0 +1,28 @@
+@import '/styles/color';
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .login {
+    width: 350px;
+    align-items: center;
+    justify-content: center;
+    @media (min-width: 768px) {
+      width: 520px;
+    }
```

```suggestion
@include mobile {}
```

만약 저희가 데스크탑, 태블릿, 모바일 기준 사이즈를 변경하게 되면 이런 코드는 일일이 변경해야할거같아요
그래서 정해진 믹스인 사용해주시면 좋을거같습니다